components/org.wso2.carbon.identity.oauth.uma.resource.service/src/test/java/org/wso2/carbon/identity/oauth/uma/resource/service/impl/ResourceServiceImplTest.java

```diff
@@ -57,14 +54,16 @@ public IObjectFactory getObjectFactory() {
     public void testRegisterResource() throws Exception {
 
         mockStatic(ResourceDAO.class);
-        assertNull(resourceService.registerResource(resource, resourceOwnerName, tenantId, consumerKey));
+        assertNull(resourceService.registerResource(resource, "admin", -1234,
```

Comment:
Change variables to constants and reuse in other places as well.

Reply:
fixed: https://github.com/wso2-extensions/identity-oauth-uma/pull/68